documentation/api/query/v4/ast.markdown

```diff
@@ -209,7 +209,18 @@ specified column is not null.
 The `to_string` function operates on timestamps and integers, allowing them to
 be formatted in a user-defined manner before being returned from puppetdb.
 Available  formats are the same as those documented for [PostgreSQL's `to_char`
-function][to-char].
+function][to-char]. For instance, to get the full lower case month name of the
+`producer_timestamp`, you can query the reports endpoint with:
+
+```
+["extract", [["function", "to_string", "producer_timestamp", "month"]]]
+```
+
+To get the last 2 digits of the year a report was submitted  from the Puppet Server:
+
+```
+["extract", [["function", "to_string", "producer_timestamp", "YY"]]]]
+```
```

Comment:
I think that having two examples for timestamps formatting, isn't necessary. I think that the second example should be about converting integers into strings. Something like this:
To convert the `uptime_seconds` fact from integer to string use the following query on the facts endpoint:
["extract", [["function", "to_string", "value"]], ["=","name", "uptime_seconds"]]

Reply:
Probably for integer conversion we need to provide a format mask? Always